source/crud/tests/v2/updateWithPipelines.yml

```diff
@@ -16,6 +16,14 @@ tests:
         arguments:
             filter: { y: 1 }
             update: [ { $replaceRoot: { newRoot: "$t" } }, { $addFields: { foo: 1 } } ]
+        result:
+          deletedCount: 0
+          insertedCount: 0
+          insertedIds: {}
+          matchedCount: 1
+          modifiedCount: 1
+          upsertedCount: 0
+          upsertedIds: {}
```

Comment:
For UpdateResult, this would be `upsertedId` singular. The plural and object type is only for BulkWriteResult where we might need a map. That said, existing `updateOne` tests that don't actually upsert (i.e. `upsertedCount` is zero) simply omit this field so I'd suggest the same here.

IIRC, even though multiple documents could match the criteria, the `matchedCount` in the UpdateResult will stop at one for `updateOne` operations.

Reply:
I have omitted the `upsertedIds` field.